app/views/all_casa_admins/casa_admins/_form.html.erb

```diff
@@ -1,17 +1,14 @@
 <%= form_for casa_admin, url: all_casa_admins_casa_org_casa_admins_path(casa_org), id: :new_casa_admin do |form| %>
   <%= render "/shared/error_messages", resource: casa_admin %>
-
-  <div class="field form-group">
+  <div class="input-style-1">
     <%= form.label :email, "Email" %>
     <%= form.text_field :email, class: "form-control" %>
   </div>
-
-  <div class="field form-group">
+  <div class="input-style-1">
     <%= form.label :display_name, "Display name" %>
     <%= form.text_field :display_name, class: "form-control" %>
   </div>
-
   <div class="actions">
-    <%= form.submit "Submit", class: "btn btn-primary" %>
+    <%= form.submit "Submit", class: "main-btn primary-btn btn-hover" %>
```

Comment:
Can you add the submit icon like the other submit buttons have. I think this should have the btn-sm class as well.

Reply:
Done